src/App.js

```diff
@@ -1,24 +1,33 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Login from './components/Login';
+import Register from './components/Register';
+import RequireAuth from './components/RequireAuth';
+import AddCarsForm from './components/cars/AddCarsForm';
+import Session from './components/session/Session';
+import MainPage from './components/cars/MainPage';
+import DeleteCar from './components/cars/DeleteCar';
+import Reservations from './components/reservations/Reservations';
+import CarDetails from './components/details/CarDetails';
+import ReserveNav from './components/reserve/ReserveNav';
+import ReserveDetails from './components/reserve/ReserveDetails';
 
 function App() {
```

Comment:
- [ ] This is the test that comes with React:

![image](https://github.com/ivangonzalez224/Tesla-car-booking-front-end/assets/57421823/ed060300-33ee-463a-ad18-3ca1a19f9aa8)

Kindly either implement it or remove it (the file) if not using it. 👍